### PR TITLE
Added project structure dialog

### PR DESCRIPTION
**Pull request by olkornii**
Signed-off-by: Oleksii Korniienko <olkornii@redhat.com>

### CODE REVIEW

File: test-project/src/test/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixturesTest/dialogs/navigation/ProjectStructureAndTipDialogs.java [hunk 1]
```diff
@@ -8,68 +9,74 @@
...
 import com.redhat.devtools.intellij.commonUiTestLibrary.fixtures.mainIdeWindow.MainIdeWindow;
 import com.redhat.devtools.intellij.commonUiTestLibrary.fixtures.mainIdeWindow.ideStatusBar.IdeStatusBar;
-import com.redhat.devtools.intellij.commonUiTestLibrary.utils.testExtension.ScreenshotAfterTestFailExtension;
 import com.redhat.devtools.intellij.commonUiTestLibrary.fixtures.dialogs.information.TipDialog;
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
+import com.redhat.devtools.intellij.commonUiTestLibrary.fixtures.dialogs.information.ProjectStructureDialog;
+import org.junit.jupiter.api.*;
```
Comment (zcervink) [inline test-project/src/test/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixturesTest/dialogs/navigation/ProjectStructureAndTipDialogs.java:18]: I suggest to remove the * notation.

Comment (olkornii) [reply test-project/src/test/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixturesTest/dialogs/navigation/ProjectStructureAndTipDialogs.java]: done

File: test-project/src/test/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixturesTest/dialogs/navigation/ProjectStructureAndTipDialogs.java [hunk 2]
```diff
@@ -8,68 +9,74 @@
...
     }
 
-    @AfterEach
-    public void closeProject() {
-        super.closeProject();
+    @AfterAll
+    public static void closeProject() {
+        LibraryTestBase.closeProject();
+    }
+
```
Comment (zcervink) [inline test-project/src/test/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixturesTest/dialogs/navigation/ProjectStructureAndTipDialogs.java:45]: I suggest to remove 1 empty line and keep only 1.

Comment (olkornii) [reply test-project/src/test/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixturesTest/dialogs/navigation/ProjectStructureAndTipDialogs.java:45]: done

**Pull request merged**